Initial draft of C++ for OpenCL documentation (#141)
KhronosGroup/OpenCL-Docs: cxx4opencl/intro.txt

// Copyright 2019 The Khronos Group. This work is licensed under a
// Creative Commons Attribution 4.0 International License; see
// http://creativecommons.org/licenses/by/4.0/

[[intro]]
== Introduction

This language is built on top of OpenCL C v2.0 and {cpp}17 enabling
most of regular {cpp} features in OpenCL kernel code. Most functionality

> from OpenCL C and {cpp} is inherited. List of restrictions is documented

Reviewer: ```suggestion
from OpenCL C and {cpp} is inherited. Restrictions are documented
```

in <<restrictions, _Restrictions_>> sections.

The main aim of this document is to describe behavior that is not

> documented in OpenCL C and {cpp}17 specification. This is mainly related

Reviewer: ```suggestion
documented in the OpenCL C and {cpp}17 specifications. This is mainly related
```
